io.asgardio.android.oidc.sdk.sample/src/main/java/io/asgardio/android/oidc/sdk/sample/SplashActivity.java

```diff
@@ -0,0 +1,41 @@
+package io.asgardio.android.oidc.sdk.sample;
```

Comment:
add license

Reply:
Resolved in https://github.com/wso2-extensions/identity-sdks-android/pull/15/commits/ca0ccdddb9b101487384388922108627685dfb13

Reply:
Add a empty line between license and the package name